atlas/src/shared/components/InfiniteVideoGrid/InfiniteVideoGrid.tsx

```diff
@@ -156,5 +161,8 @@ const StyledVideoPreviewBase = styled(VideoPreviewBase)`
   margin: 0 auto;
   width: 100%;
 `
+const StyledGrid = styled(Grid)`
+  padding-top: ${sizes.b2}px;
```

Comment:
Why this change? In the design there is a 16px margin between title and the grid, like it is right now